packages/frontend/core/src/modules/multi-tab-sidebar/multi-tabs/tabs/chat.tsx

```diff
@@ -27,6 +34,7 @@ const EditorChatPanel = ({ editor }: SidebarTabProps) => {
 
   if (editor !== chatPanelRef.current?.editor) {
     (chatPanelRef.current as ChatPanel).editor = editor;
```

Comment:
can we infer doc from editor instance?

Reply:
yes, but when doc switched, chat panel don't get informed.

Reply:
I guess you need to hook into doc change event somehow. There could be other cases that changes the active doc

Reply:
okay， I'll change it later